configure.py

```diff
@@ -529,7 +529,6 @@ def find_headers(repodir, excluded_dirs):
     'test/boost/bptree_test',
     'test/boost/btree_test',
     'test/boost/radix_tree_test',
-    'test/boost/double_decker_test',
```

Comment:
nit: it's better to remove the other entry, as this is the local cluster of collections tests -  bptree, btree, radix_tree and double_decker

Reply:
ahh, didn't realize this!